src/Illuminate/Mail/Message.php

```diff
@@ -24,6 +25,8 @@ class Message
     /**
      * CIDs of files embedded in the message.
      *
+     * @deprecated Will be removed in a future Laravel version.
+     *
      * @var array
      */
     protected $embeddedFiles = [];
```

Comment:
This is not used anywhere, so I deprecated it. Happy for it to be reverted if it has a purpose.

Reply:
// cc @driesvints

Reply:
Sounds good